runtime/compiler/z/runtime/Math.m4

```diff
@@ -832,6 +832,7 @@ ifdef({ASM_J9VM_JIT_32BIT_USES64BIT_REGISTERS},{dnl
 ifdef({J9ZOS390},{dnl
 ifdef({TR_HOST_64BIT},{dnl
 ZZ 64bit XPLINK doesn't need call descriptors
+    END
```

Comment:
Perhaps a better solution is to move this out of the 64-bit conditional, without the newline (like the change for `PicBuilder.m4`)? This file ends with two newlines, which I expect will be problematic with Open XL.

Reply:
@Deigue Please address the concerns from Keith, if we need code-change - will assist to get one more sanity build internally

Reply:
The generated output of this file compiled fine with Open XL so thats odd ... but yes, I can move the "END" out ... figured it was a bit odd just having what looks like a comment in the 64BIT clause alone.

Reply:
Comment does make sense though highlighting why call descriptor was omitted